.travis.yml

```diff
@@ -87,32 +87,28 @@ matrix:
         - env: CONDA_DEPENDENCIES=$CONDA_REQUIRED_DEPENDENCIES
                PIP_DEPENDENCIES=''
 
-        # Test with Astropy dev version and the latest Python
+        # Test with other Python versions
         - stage: Comprehensive tests
-          env: ASTROPY_VERSION=development
-               EVENT_TYPE='pull_request push cron'
+          env: PYTHON_VERSION=3.7
 
         # Test with Astropy LTS version and older numpy versions
-        # astropy >= 3.1 requires numpy >= 1.13
         - stage: Comprehensive tests
           env: PYTHON_VERSION=3.5
-               NUMPY_VERSION=1.11
+               NUMPY_VERSION=1.13
                ASTROPY_VERSION=lts
                PIP_DEPENDENCIES='pytest<3.10 scikit-image>0.14.1'
+               EVENT_TYPE='pull_request push cron'
 
+        # Test with other numpy versions
         - stage: Comprehensive tests
-          env: NUMPY_VERSION=1.12 ASTROPY_VERSION=lts
-               PIP_DEPENDENCIES='pytest<3.10 scikit-image>0.14.1'
+          env: PYTHON_VERSION=3.7
+               NUMPY_VERSION=1.15
 
+        # Test with Astropy dev version
         - stage: Comprehensive tests
-          env: NUMPY_VERSION=1.13 ASTROPY_VERSION=lts
-               PIP_DEPENDENCIES='pytest<3.10 scikit-image>0.14.1'
+          env: ASTROPY_VERSION=development
                EVENT_TYPE='pull_request push cron'
 
-        # Test with other Python and numpy versions
-        - stage: Comprehensive tests
-          env: PYTHON_VERSION=3.7
-
         # Test with numpy pre-release version
         - stage: Comprehensive tests
           env: NUMPY_VERSION=prerelease
```

Comment:
I think there is no need for prerelease testing. Either test with numpy dev, or trust that most of the things photutils will run into will be captured by the astropy tests against numpy dev.

Reply:
Ah, OK.  I'll change this to `NUMPY_VERSION=dev`. Thanks!

Reply:
Done in 3efaafe47b371182d54744b75f202f6557439270.

Reply:
I don't want to be a pain with this, but pairing the numpy dev and astropy dev might make sense now given that Marten has added the works on `__array_function__`

Reply:
As long as if there are failures that we can clearly tell if astropy dev or numpy dev was the cause.

Reply:
I guess we'll be able to tell but I'm not sure (if it's a numpy dev issue I suppose there'll be issues on the core CI, too).

Reply:
My other concern is numpy dev could break something in astropy dev that has nothing to do with `photutils`.  The `photutils` test would fail, but the problem is upstream (and presumably will be caught in astropy CI).

Reply:
yes, that's very likely to happen.

Reply:
OK, I'll leave this for now.  Thanks! (and you weren't being a pain at all!  I appreciate you looking at this!)